README.md

```diff
@@ -76,5 +76,23 @@ Automatically fix linting issues:
 ./gradlew generateSwaggerCode
 ```
 
+### Running SourceClear locally
```

Comment:
Thank you for documenting this!!

Reply:
My pleasure! I will have a follow-on to get this in LZS too, and will get it in WSM.